src/components/ModalContent/ModalContent.test.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { fireEvent } from '@folio/jest-config-stripes/testing-library/react';
+import { MemoryRouter } from 'react-router-dom';
+import ModalContent from './ModalContent';
+import { renderWithIntl } from '../../../test/jest/helpers';
+import '../../../test/jest/__mock__';
+
+const stripesMock = {
+  hasPerm: jest.fn().mockReturnValue(true),
+};
+
+const itemMock = {
+  title: 'Sample Title',
+  barcode: '123456789',
+  materialType: {
+    name: 'Book',
+  },
+};
+
+describe('ModalContent', () => {
+  it('renders with correct props and messages', () => {
```

Comment:
Let's not test 3 different cases in one `it`

Please split this test into multiple cases:
1. The modal content is rendered
2. when the cancel button is pressed the `onCancel` callback is called
3. when the confirm button is pressed the `onConfirm` callback is called

For this it will be useful to move
```
renderWithIntl(
      <MemoryRouter>
        <ModalContent
          stripes={stripesMock}
          item={itemMock}
          status={status}
          requestsUrl={requestsUrl}
          onCancel={onCancelMock}
          onConfirm={onConfirmMock}
          itemRequestCount={itemRequestCount}
        />
      </MemoryRouter>
    );
```
into a separate `renderModalContent` function

Reply:
Thanks for that! Done!